Sources/Widgets/Widgets3D/ResliceCursorWidget/example/controlPanel.html

```diff
@@ -18,9 +18,24 @@
     </td>
   </tr>
   <tr>
+    <td>Slab Mode :</td>
     <td>
-      <button id="buttonReset">Reset views:</button>
+      <select id="slabMode">
+        <option id="slabModeMin">MIN</option>
+        <option id="slabModeMax">MAX</option>
+        <option id="slabModeMean" selected="selected">MEAN</option>
+        <option id="slabModeSum">SUM</option>
+      </select>
     </td>
   </tr>
-
+  <tr>
+    <td>Slab Number of Slices :</td>
+    <td><input id='slabNumber' type="range" min="1" max="10" step="1" value="1" style="width: 100px;"/></td>
```

Comment:
Can you please increase max value to 100 (or ideally the input volume diagonal length)

Reply:
I modified that set initial max value to 100. After loading volume, set as input volume dimension diagonal length.

Reply:
This is great ! Thanks, I'm going to merge it. Great job, thanks for your contribution !

Concerning the performance, this is what I feared, as the number of slices increases, the performance get poor. I do not think there is much we can do with ImageReslice (CPU).

Doing it on the GPU would definitely increase performance. The idea would be to replace ImageReslice + ImageMapper with VolumeMapper (which also has slab support).